index.js

```diff
@@ -160,7 +156,7 @@ module.exports = {
         // so it has to be disabled
         '@typescript-eslint/no-var-requires': 'off',
 
-        '@typescript-eslint/no-explicit-any': 'off',
```

Comment:
Why did we change to just warn?

Reply:
Should we change it to show error instead? @wildtron

Reply:
Try changing this to error, I think bex will fail to build.

Reply:
warn is to prevent the devs from pushing, but to allow coding while not yet perfectly knowing the type.

Reply:
I think we should, since that's one of our main reasons to use TS 🤔